internal/backends/qt/qt_widgets/groupbox.rs

```diff
@@ -120,10 +142,14 @@ impl Item for NativeGroupBox {
     }
 
     fn layout_info(self: Pin<&Self>, orientation: Orientation, _window: &WindowRc) -> LayoutInfo {
+        let text: qttypes::QString = self.title().as_str().into();
+
+        let size = minimum_group_box_size(text);
+
         LayoutInfo {
             min: match orientation {
-                Orientation::Horizontal => self.native_padding_left() + self.native_padding_right(),
-                Orientation::Vertical => self.native_padding_top() + self.native_padding_bottom(),
+                Orientation::Horizontal => size.width as f32,
+                Orientation::Vertical => size.height as f32,
```

Comment:
After this change the `native_padding_XXX` fields are unused. Are you sure that they should be ignored?

Reply:
I think they ate still used by the layout on the .slint side

Reply:
@ogoffart: I have not seen where those properties are used, but I did not dare remove them completely :-)

Reply:
>  I have not seen where those properties are used, but I did not dare remove them completely

Bad anwser :-)
The goal is always to remove as much as possible. 
 If one does not know what something is for, one should try to quickly find out with a rapid search.

In this case, the native_padding_XXX properties are used in a layout in the GroupBox implementation so that the inner widget occupies the right position.